modules/standard/Assert.chpl

```diff
@@ -2,69 +2,25 @@
  * Copyright 2020 Hewlett Packard Enterprise Development LP
  * Copyright 2004-2019 Cray Inc.
  * Other additional copyright holders may be indicated within.
- * 
+ *
  * The entirety of this work is licensed under the Apache License,
  * Version 2.0 (the "License"); you may not use this file except
  * in compliance with the License.
- * 
+ *
  * You may obtain a copy of the License at
- * 
+ *
  *     http://www.apache.org/licenses/LICENSE-2.0
- * 
+ *
  * Unless required by applicable law or agreed to in writing, software
  * distributed under the License is distributed on an "AS IS" BASIS,
  * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
  * See the License for the specific language governing permissions and
  * limitations under the License.
  */
 
-
-/*
-  Support for simple assert() routines.
-
-  .. note:: All Chapel programs automatically ``use`` this module by default.
-            An explicit ``use`` statement is not necessary.
-
-  In the current implementation, these asserts never become no-ops.  That is,
-  using them will always incur execution-time checks.
-
-*/
 module Assert {
```

Comment:
I think that @ben-albrecht (?) has had some text to use for modules that are being deprecated so that if one starts leafing through different versions of a module in the online chpldocs, they will see a description of it disappearing rather than having it simply not be there in some release (?).

Reply:
https://github.com/chapel-lang/chapel/blob/master/doc/rst/developer/bestPractices/Deprecation.rst

```
    Foo module short description.

    Warning

    This module has been deprecated - please use :mod:`bar` instead.

    Foo module long description.
```